main.py

```
@@ -0,0 +1,104 @@
+    """Ensure MediaPipe task model files are present in models/."""
+    os.makedirs(config.MODELS_DIR, exist_ok=True)
+    model_urls = {
+        config.FACE_MODEL_PATH: "https://storage.googleapis.com/mediapipe-models/face_landmarker/face_landmarker/float16/1/face_landmarker.task",
+        config.HAND_MODEL_PATH: "https://storage.googleapis.com/mediapipe-models/hand_landmarker/hand_landmarker/float16/1/hand_landmarker.task",
+        config.POSE_MODEL_PATH: "https://storage.googleapis.com/mediapipe-models/pose_landmarker/pose_landmarker_lite/float16/1/pose_landmarker_lite.task"
+    }
+    for path, url in model_urls.items():
+        if not os.path.exists(path):
+            try:
+                print(f"[INIT] Downloading vision model: {os.path.basename(path)}...")
+                urllib.request.urlretrieve(url, path)
```

Ruff (0.16.3)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)